assets/js/hooks/index.ts

```diff
@@ -16,6 +17,15 @@ export const Flash = {
   },
 } as PhoenixHook<{ timer: ReturnType<typeof setTimeout> }>;
 
+export const Tooltip = {
+  mounted() {
+    let content = this.el.ariaLabel;
+    tippy(this.el, {
+      content: `${content}`,
```

Comment:
Why is the `content` variable inside a string interpolation? Is this to ensure it's not undefined or null?

If that's the case, can you instead _not_ call `tippy` if the `ariaLabel` is falsey, and call `console.debug` with a message saying that the Tooltip hook was mounted but the element doesn't have a label.

Also I don't see an `destroyed` function, are we sure that `tippy` doesn't have/need an destroy/cleanup functions?

Reply:
@elias-ba I have made the changes here: https://github.com/OpenFn/Lightning/pull/1017/commits/f1675e007fbc8bf291b1c1e4e7c40f839b4af06b